Mathlib/Data/ENNReal/Basic.lean

```diff
@@ -710,8 +710,28 @@ def evalENNRealOfNNReal : PositivityExt where eval {u α} _zα _pα e := do
     | _ => pure .none
   | _, _, _ => throwError "not ENNReal.ofNNReal"
 
+private lemma ennreal_one_pos : (0 : ℝ≥0∞) < 1 := zero_lt_one
+private lemma ofNat_pos {n : ℕ} [n.AtLeastTwo] : 0 < (no_index OfNat.ofNat n : ℝ≥0∞) :=
```

Comment:
Do you need the `no_index` if you're not using it with `simp`?

Reply:
Copy pasta :)